lib/pbench/test/unit/server/query_apis/test_iteration_samples.py

```diff
@@ -0,0 +1,328 @@
+                                "sum_other_doc_count": 0,
+                                "buckets": [
+                                    {
+                                        "key": "throughput",
+                                        "doc_count": 50,
+                                        "title": {
+                                            "doc_count_error_upper_bound": 0,
+                                            "sum_other_doc_count": 0,
+                                            "buckets": [
+                                                {
+                                                    "key": "Gb_sec",
+                                                    "doc_count": 50,
+                                                    "uid": {
+                                                        "doc_count_error_upper_bound": 0,
+                                                        "sum_other_doc_count": 0,
+                                                        "buckets": [
+                                                            {
+                                                                "key": "client_hostname:127.0.0.1-server_hostname:127.0.0.1-server_port:20010",
+                                                                "doc_count": 25,
+                                                            },
+                                                            {
+                                                                "key": "client_hostname:all-server_hostname:all-server_port:all",
+                                                                "doc_count": 25,
+                                                            },
+                                                        ],
+                                                    },
+                                                }
+                                            ],
+                                        },
+                                    }
+                                ],
+                            },
+                        }
+                    ],
+                },
+            },
+        }
+        index = self.build_index(
+            server_config, self.date_range(self.payload["start"], self.payload["end"])
+        )
+
+        if HeaderTypes.is_valid(build_auth_header["header_param"]):
+            expected_status = HTTPStatus.OK
+        else:
+            expected_status = HTTPStatus.FORBIDDEN
+
+        response = query_api(
+            self.pbench_endpoint,
+            self.elastic_endpoint,
+            self.payload,
+            index,
+            expected_status,
+            json=response_payload,
+            status=HTTPStatus.OK,
+            headers=build_auth_header["header"],
+            scroll=True,
+        )
+        if expected_status == HTTPStatus.OK:
+            res_json = response.json
+            expected_result = {
```

Comment:
How is this value of `expected_result` different from the value of `response_payload` set at line 46?  Do we really want to maintain independent respective definitions for them?